jane-php/json-schema/7.0/bin/jane-json-schema-generate

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env php
+<?php
+
+require __DIR__ . '/../vendor/autoload.php';
+
+use Jane\Component\JsonSchema\Console\Command\GenerateCommand;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Output\NullOutput;
+use Jane\Component\JsonSchema\Console\Loader\ConfigLoader;
+use Jane\Component\JsonSchema\Console\Loader\SchemaLoader;
+
+$command = new GenerateCommand(new ConfigLoader(), new SchemaLoader());
```

Comment:
why make it special and not a regular Symfony command registered by the bundle?

Reply:
This is unexpected to me, ppl are used to run commands through the `bin/console` script, so if that's possible, that's better indeed

Reply:
Since that library isn't a bundle it was not made that way, but I'll add a bundle to handle that :+1:

Reply:
I removed the json-schema / open-api library updates, let's focus on the AutoMapper bundle, the rest will come later when I have a bundle for each of them :)